ds3-sdk-integration/src/test/java/com/spectralogic/ds3client/integration/PutJobManagement_Test.java

```diff
@@ -0,0 +1,337 @@
+
+    @Test
+    public void modifyJobName() throws IOException, SignatureException, XmlProcessingException {
+        final String bucketName = "test_modify_job_name";
+        try {
+            client.putBucket(new PutBucketRequest(bucketName));
+
+            final List<Ds3Object> objects = Lists.newArrayList(new Ds3Object("test",2));
+
+            final Ds3ClientHelpers.Job job =
+                    wrap(client).startWriteJob(bucketName, objects);
+
+            client.modifyJobSpectraS3(new ModifyJobSpectraS3Request(job.getJobId())
+                    .withName("newName"));
+
+            final GetJobSpectraS3Response response = client
+                    .getJobSpectraS3(new GetJobSpectraS3Request(job.getJobId()));
+
+            assertThat(response.getMasterObjectListResult().getName(), is("newName"));
+
+        } finally {
+            deleteAllContents(client, bucketName);
+        }
+    }
+
+    @Test
+    public void modifyJobCreationDate() throws IOException, SignatureException, XmlProcessingException {
+        final String bucketName = "test_modify_job_creation_date";
+        try {
+            client.putBucket(new PutBucketRequest(bucketName));
+
+            final List<Ds3Object> objects = new ArrayList<>();
+            final Ds3Object obj = new Ds3Object("test", 2);
+            objects.add(obj);
+
+            final Ds3ClientHelpers.Job job =
+                    wrap(client).startWriteJob(bucketName, objects);
+            final GetJobSpectraS3Response jobResponse = client
+                    .getJobSpectraS3(new GetJobSpectraS3Request(job.getJobId()));
+
+            final Date originalDate = jobResponse.getMasterObjectListResult().getStartDate();
+            final Date newDate = new Date(originalDate.getTime() - 1000);
+
+            client.modifyJobSpectraS3(new ModifyJobSpectraS3Request(job.getJobId())
+                    .withCreatedAt(newDate));
+
+            final GetJobSpectraS3Response responseAfterModify = client
+                    .getJobSpectraS3(new GetJobSpectraS3Request(job.getJobId()));
+
+            assertThat(responseAfterModify.getMasterObjectListResult().getStartDate(), is(newDate));
+
+        } finally {
+            deleteAllContents(client, bucketName);
+        }
+    }
+
+    @Test
+    public void cancelJob() throws IOException, SignatureException, XmlProcessingException {
+        final String bucketName = "test_cancel_job";
+        try {
+            client.putBucket(new PutBucketRequest(bucketName));
+
+            final List<Ds3Object> objectsOne = new ArrayList<>();
+            final Ds3Object obj = new Ds3Object("testOne", 2);
+            objectsOne.add(obj);
+
+            final Ds3ClientHelpers.Job job =
+                    wrap(client).startWriteJob(bucketName, objectsOne);
+
+            final CancelJobSpectraS3Response response = client
+                    .cancelJobSpectraS3(new CancelJobSpectraS3Request(job.getJobId()));
+            response.checkStatusCode(204);
```

Comment:
The response generation already does this.  Here you should be using an assert and then getting the status code.